python/tank_vendor/shotgun_authentication/login_dialog.py

```diff
@@ -226,8 +226,7 @@ def _ok_pressed(self):
             # We need a two factor authentication code, move to the next page.
             self.ui.stackedWidget.setCurrentWidget(self.ui._2fa_page)
         except Exception, e:
-            self._set_error_message(self.ui.message, e[0])
-            raise
+            self._set_error_message(self.ui.message, e)
```

Comment:
Exceptions are properly filtered at the source now, no need to escape extract part of it.